crates/driver/src/domain/competition/auction.rs

```diff
@@ -418,6 +445,26 @@ impl From<eth::U256> for Price {
     }
 }
 
+lazy_static::lazy_static! {
+    static ref UNIT: num::BigInt = num::BigInt::from(1_000_000_000_000_000_000_u128);
+}
+
+/// The price of a token normalized to native token price.
+#[derive(Debug, Clone)]
+pub struct NormalizedPrice(pub BigRational);
```

Comment:
I still don't really understand the need for normalized prices. Wouldn't it be equivalent to do all the computations using the regular prices and only apply some conversion to the final result?

Reply:
moved normalization to the callsite